frontend/src/features/projects/list/ProjectApprovalRequestListView.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { ProjectListView } from '.';
+
+/**
+ * A list view paging table to search for projects that have been submitted for assessment.
+ * @returns ProjectApprovalRequestListView component.
+ */
+export const ProjectApprovalRequestListView = () => {
+  console.info('test');
```

Comment:
this should probably be removed.

Reply:
also is this a bug fix? this seems like new functionality.

Reply:
This was to fix bugs within CITZ PIMS.